ui/src/app/components/sidenav-rule/sidenav-rule.component.html

```diff
@@ -0,0 +1,46 @@
+<div class="sidenav-container">
+  <div class="sidenav-header">
+    <span class="header-title">Add rule</span>
+    <button class="close-button" mat-icon-button color="primary" (click)="closeSidenav()">
+      <mat-icon class="close-icon">close</mat-icon>
+    </button>
+  </div>
+  <div class="form-container">
+    <form [formGroup]="ruleForm">
+      <h3>Rule info</h3>
+      <mat-form-field appearance="outline">
+        <mat-label>Rule name</mat-label>
+        <input matInput class="input-field" formControlName="ruleName" />
+        <mat-hint>Max. 60 characters, and starts with a letter.</mat-hint>
+      </mat-form-field>
+
+      <h3>Rule definition</h3>
+      <mat-form-field appearance="outline">
+        <mat-label>Rule type</mat-label>
+        <mat-select
+          #ruleType
+          matSelect
+          class="input-field"
+          formControlName="ruleType"
+          required="true"
+        >
+          <mat-option value="globalDataType">Change global data type</mat-option>
+          <mat-option value="localDataType">Change one table data type</mat-option>
```

Comment:
Change one table data type functionality is not working

Reply:
This feature was not there in the old UI and so it is not implemented yet in the new UI. It can go into the backlog list.